llvm/test/CodeGen/RISCV/rvv/vlopt-same-vl.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -mtriple=riscv64 -mattr=+v -riscv-enable-vl-optimizer \
+; RUN:   -verify-machineinstrs -debug-only=riscv-vl-optimizer -o - 2>&1 %s | FileCheck %s 
+
+; REQUIRES: asserts
+
+define <vscale x 4 x i32> @same_vl_imm(<vscale x 4 x i32> %passthru, <vscale x 4 x i32> %a, <vscale x 4 x i32> %b) {
```

Comment:
Please add a comment which explains *why* you're checking for the particular debug output.

Reply:
Though, honestly, this seems like borderline bad practice.  I would be fine with this landing without a dedicated test since it is NFC and "only" improves compile time.

Reply:
Please let me know what you prefer.